cmd/config-current.go

```diff
@@ -359,6 +385,11 @@ func newServerConfig() *serverConfig {
 		},
 		KMS:    crypto.KMSConfig{},
 		Notify: notifier{},
+		Compression: compressionConfig{
+			Enabled:    false,
+			Extensions: []string{".txt", ".log", ".csv", ".json"},
```

Comment:
can we have global values for defaults which can be centrally edited in future?

Reply:
Done